index.md

```diff
@@ -24,6 +24,10 @@ constitute the Trixi framework on [GitHub](https://github.com/orgs/trixi-framewo
 
   Convert output files generated with Trixi.jl to VTK
 
+* [**Libtrixi**](https://github.com/trixi-framework/libtrixi)
+
+  Use [Trixi.jl](https://github.com/trixi-framework/Trixi.jl) from C/C++/Fortran
+
 * [**SmartShockFinder.jl**](https://github.com/trixi-framework/SmartShockFinder.jl)
 
   Create troubled cell indicators for Trixi.jl using artificial neural networks
```

Comment:
I'm not sure whether we should really advertise packages with READMEs like "anything can change anytime, this is far from being stable/usable" should be advertised in the same way as Trixi.jl, Trixi2Vtk.jl etc.

Reply:
Yeah, I wasn't sure about this either, which is I requested a review from you. IMHO libtrixi is in "research software alpha" stage, i.e., basic functionality is there and working, it is usable, but there might be changes and stuff can still break.

However, there's also so much effort already been put into it, I wanted to put it somewhere more prominently.

What if we change the README statement to something like
> Note: This project is in early stage and the API is still under development. Basic functionality, however, is already implemented and actively tested.

Would that help?

Reply:
That looks definitely better. We could also add  comment to the website entries but I would leave this decision up to you.